src/test/Testcafe/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "local" :
+    {
+    "baseUrl": "http://altinn3.no",
+    },
+    "dev" :
+    {
+        "baseUrl: "base for dev",
+    }
+    "uat" :
+    {
+        "baseUrl": "base for uat",
+    }
+}
```

Comment:
This .json isn't valid:  
- Missing `,` after "dev".

`,` after will fail validation:
- local.baseUrl
- dev.baseUrl
- uat.baseUrl

Reply:
fixed invalid json